### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by weblate**

### CODE REVIEW

**Comment by aparcar** [comment]
@jow- this is a bit of a bigger PR as it add PO files to all languages, making the translations consistent. This is automatically added to the weblate interface so external people are motivated to do some translating. Merge if you like the idea.

File: applications/luci-app-commands/po/de/commands.po [hunk 1]
```diff
@@ -100,7 +100,7 @@ msgstr "Herunterladen"
 
 #: applications/luci-app-commands/luasrc/view/commands.htm:119
 msgid "Download execution result"
-msgstr ""
+msgstr "Ausführungsergebnis herunterladen"
```
Comment (jow-) [inline applications/luci-app-commands/po/de/commands.po]: Programmausgabe herunterladen

File: applications/luci-app-commands/po/de/commands.po [hunk 2]
```diff
@@ -128,11 +128,11 @@ msgstr "Ausführen"
 
 #: applications/luci-app-commands/luasrc/view/commands_public.htm:41
 msgid "Standard Error"
-msgstr ""
+msgstr "Standardfehler"
```
Comment (jow-) [inline applications/luci-app-commands/po/de/commands.po]: Fehlerausgabe

**Comment by jow-** [comment]
Note that Weblate added languages which are not yet covered in luci.mk

**Comment by feckert** [comment]
@aparcar How can I add missing apps in weblate?
Specifically I am talking about luci-app-mwan3?

**Pull request merged**